lib/tasks/missing_version_timestamps.rake

```diff
@@ -0,0 +1,133 @@
+desc "One off tasks to fix missing submission version timestamps"
+namespace :missing_version_timestamps do
+
+  desc "When the latest record missed the required timestamp"
+  task tranch1: :environment do
+    SubmissionVersion.where(created_at: nil)
```

Comment:
This is redundant, right? Unless it should be a guard clause?

Reply:
yes removed